description.md

```diff
@@ -11,6 +11,25 @@ Each section details how a specific type of content is to be developed, stored a
 
 ### Reading
 
+The reading content type is the most common form of storing educational material (or information, in general).
+As the name suggests the content is developed to be read, therefore it comes in the form of text.
+
+To develop reading content we use the [Markdown syntax](https://www.markdownguide.org/basic-syntax) and the following guidelines:
+
+- the heading level of a title is given by the number of `#`s that precede it.
+For example, the title of a document is preceded by a single `#`, whereas the chapters are preceded by two.
+- each sentence is written on a new line.
+- paragraphs are separated by an empty line.
+- links should be used as hyperlinks by using the `[]()` syntax as in [this example](https://theuselessweb.com/).
+
```

Comment:
There are a few more rules enforced by our linters such as leaving blank lines before and after lists, using the same number for every item of a numbered list, such as:

```
1. first item

1. second item
```

But listing them exhaustively would be pointless. I'm not sure which ones are relevant to this section. Contributors will "discover" them anyway when they want to raise a PR, but not knowing about them beforehand will surely create frustration.

A good thing you can do is to give a link to [super-linter](https://github.com/super-linter/super-linter#readme) so people can at least know what behemoth lies ahead of them.

Reply:
@teodutu I added a couple of sentences.